sqlserver/assets/configuration/spec.yaml

```diff
@@ -131,13 +131,15 @@ files:
         display_default: true
     - name: database_metrics
       description: |
-        Configure the collection of database metrics
+        Configure the collection of database metrics. Requires `dbm: true`.
```

Comment:
database_metrics shouldn't require dbm

Reply:
Ah, made the wrong assumption, will fix.